tools/custom-organization-policy-library/README.md

```diff
@@ -47,12 +55,57 @@ make build
 make constraints                    Build constraint based on input configuration
 make policies                       Build constraint based on input configuration
 make build                          Build constraint and policies based on input configuration
+make deploy-constraint              Build deploy constraint based on input configuration
+make deploy-policy                  Build deploy policy based on input configuration
+make deploy                         Build deploy based on input configuration
+make clean                          Get rid of object and execuatable files
 make config                         Inline Rego rules into constraint templates
 make format                         Format yaml and starlark files
 make help                           Prints help for targets with comments
 make test                           Test custom organization policies
 ```
 
+## Using dry-run mode
+
+To use dry run mode, Users will need to go to the **Organization policies** page in **Google cloud console**.
```

Comment:
Wrong here, @edyw did already the code to geneate dryrun in values.yaml
```
#@data/values
---
organization: '11111111'
bundles:
  pci-dss: false
  cis: false
dryrun: false
```
Set the boolean dryrun to true and regenerate the policies or something like that